Userland/Libraries/LibMedia/FFmpeg/FFmpegVideoDecoder.cpp

```diff
@@ -0,0 +1,207 @@
+/*
+ * Copyright (c) 2024, Gregory Bertilson <zaggy1024@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibCore/System.h>
+#include <LibMedia/VideoFrame.h>
+
+#include "FFmpegHelpers.h"
+#include "FFmpegVideoDecoder.h"
+
+namespace Media::FFmpeg {
+
+static AVPixelFormat negotiate_output_format(AVCodecContext*, [[maybe_unused]] AVPixelFormat const* formats)
```

Comment:
yikes, is this one of those null terminated arrays? does the lib not provide any length info?

Reply:
They don't mention any alternative in the docs for the function, so I guess this is the best we got.